source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -129,18 +129,18 @@ Moves modes define how the "move" encoder (the one on the left) reacts when used
 
 Here is how to tell use each modes:
 
-+----------+-------------------------------------------------------------+
-| Mode     |                                                             |
-+==========+=============================================================+
-| Beat     | This mode is enable if no other mode are enabled            |
-+----------+-------------------------------------------------------------+
-| Grid     |  This mode is enabled while GRID is maintained pressed      |
-+----------+-------------------------------------------------------------+
-| BPM      | This mode is enabled while :hwlabel:`SHIFT` + GRID is       |
-|          | maintained pressed                                          |
-+----------+-------------------------------------------------------------+
-| Keyboard | This mode is enabled while STEM is maintained pressed       |
-+----------+-------------------------------------------------------------+
++----------+---------------------------------------------------------------------+
+| Mode     |                                                                     |
++==========+=====================================================================+
+| Beat     | This mode is enable if no other mode are enabled                    |
++----------+---------------------------------------------------------------------+
+| Grid     |  This mode is enabled while :hwlabel:`GRID` is maintained pressed   |
++----------+---------------------------------------------------------------------+
+| BPM      | This mode is enabled while :hwlabel:`SHIFT` + :hwlabel:`GRID` is    |
+|          | maintained pressed                                                  |
```

Comment:
```suggestion
| BPM      | This mode is enabled while :hwlabel:`SHIFT` + :hwlabel:`GRID` are   |
|          | held down                                                           |
```

Reply:
just sounds strange. maybe check Deepl or some other translator for better wording.

Here and below

Reply:
Updated